org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/contentassist/SnippetCompletionProposal.java

```diff
@@ -157,7 +157,7 @@ private static boolean needsPublic(ICompilationUnit cu, CompletionContext comple
 						}
 						if (node instanceof CompletionOnSingleNameReference) {
 							CompilationUnit ast = CoreASTProvider.getInstance().getAST(cu, CoreASTProvider.WAIT_YES, null);
-							if (monitor.isCanceled()) {
```

Comment:
In this file, what about https://github.com/eclipse/eclipse.jdt.ls/blob/4a1695a5722655ccbb6ffbdc438c139b352e5649/org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/contentassist/SnippetCompletionProposal.java#L435-L436 ?

Reply:
indeed. I fixed it in an earlier attempt, but then missed it in this one